src/coreclr/vm/syncblk.cpp

```diff
@@ -2334,6 +2330,17 @@ void ObjHeader::PulseAll()
 //
 // ***************************************************************************
 
+#endif // !DACCESS_COMPILE
+
+// the DAC needs to have this method available
+PTR_Thread AwareLock::GetHoldingThread()
```

Comment:
Nit: Would be good to replace the two remaining references to this function outside of the DAC, from `threads.cpp` in stress logs, to report the holding thread ID instead.